Testing/functional/e2e/package.json

```diff
@@ -6,14 +6,14 @@
     "scripts": {
         "launch": "cypress open",
         "e2e": "cypress run",
-        "lint": "eslint cypress --ext .js --max-warnings 0",
-        "lintfix": "eslint cypress --ext .js --fix"
+        "lint": "eslint cypress --ext .js --max-warnings 0 && prettier --check cypress",
```

Comment:
Can we use eslint config to pickup up prettier preferences instead of having to run the two commands?